pyqt-apps/siriushla/si_ap_fofb/main.py

```diff
@@ -362,6 +362,37 @@ def _setupDetailsWidget(self):
                 glay2.addWidget(lbl, 3, 0)
                 glay2.addLayout(hbox, 3, 1, 1, 2)
 
+                ld_acc_filter = QLabel(
+                    'Filter: ', self, alignment=Qt.AlignRight | Qt.AlignVCenter)
+                pref = self.devpref
+                visrule = (
+                    '[{"name": "VisRule", "property": "Visible", ' +
+                    '"expression": "ch[0] == 2", "channels": ' +
+                    '[{"channel": "' +
+                    pref.substitute(propty='FOFBAccFilter-Sel') +
+                    '", "trigger": true}]}]')
+                sel_acc_filter = SiriusEnumComboBox(
+                    self, pref.substitute(propty = 'FOFBAccFilter-Sel'))
+                sts_acc_filter = SiriusLabel(
+                    self, pref.substitute(propty='FOFBAccFilter-Sts'))
```

Comment:
There was something weird with the "Custom" option where it extended the combo box beyond its original size. I don't know what caused it...

Reply:
the previous commit fix this :)